tests/unittests/OperatorTest.cpp

```diff
@@ -3005,6 +3005,78 @@ TEST_P(OperatorTest, QuantizedArgMaxNoKeepDim) {
   testArgMaxNoKeepDim<int8_t>(bindings_, mod_, F_, EE_, ElemKind::Int8QTy);
 }
 
+TEST_P(OperatorTest, FloatArgMaxNoKeepDimWithAxis1) {
+  CHECK_IF_ENABLED();
+
+  auto *input = createPlaceholderConditionallyQuantized(
```

Comment:
nit: `createPlaceholderConditionallyQuantized()` is usually called when using a helper where the ElemKind is specified through a function call. Here we should just use `mod_.createPlaceholder().`

Reply:
Done.